components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/IMessageEncoder.cs

```diff
@@ -43,17 +43,19 @@ public interface IMessageEncoder {
         /// </summary>
         /// <param name="message"></param>
         /// <param name="maxMessageSize"></param>
+        /// <param name="useReversibleEncoding"></param>
```

Comment:
Please don't follow bad practices, rather remove the <param /> comment elments than adding a new empty one

Reply:
Agreed! I have fixed it. Removing comments causes build error. Filled out all params instead.